e2e-tests/production-runtime/cypress/integration/global-styles.js

```diff
@@ -0,0 +1,19 @@
+const rgbBlack = `rgb(0, 0, 0, 0.8)` // css jQuery method returns rgb
```

Comment:
One note here: this is the value in _my_ browser, but I wonder if there's a better way to test this? Basically I want to assert that the text is colored, and it's not colored the _default_.

Reply:
Can we test exactly for colors defined in .css files?

Reply:
Yeah - except the browser tweaks them, ha. So rgb(255, 0, 0) just adds an alpha channel I believe. Let me test that!

Reply:
alternatively we could use other css properties - like `z-index` which has exact integer values (no different formats, no `px` etc)?